sdk/core/azure-core/src/main/java/com/azure/core/http/policy/HttpLogOptions.java

```diff
@@ -183,4 +184,25 @@ public HttpLogOptions setApplicationId(final String applicationId) {
         }
         return this;
     }
+
+    /**
+     * Gets flag to allow pretty printing of JSON message bodies.
+     *
+     * @return true if pretty printing of JSON message bodies is allowed.
+     */
+    public boolean isPrettyPrintJson() {
```

Comment:
I know the original issue stated to make the pretty print flag settable, but given that there are other formats such as `XML` could this be called `prettyPrintBody`?

Reply:
Renamed to `prettyPrintBody`